formats/cbor/commonMain/src/kotlinx/serialization/cbor/ByteString.kt

```diff
@@ -0,0 +1,24 @@
+package kotlinx.serialization.cbor
+
+import kotlinx.serialization.SerialInfo
+
+/**
+ * Specifies that a [ByteArray] shall be encoded/decoded as CBOR major type 2: a byte string.
```

Comment:
I would also make a note that this annotation does not affect types other than byte array